stderr of test 'pyUDF-crash.Bug-6508` in directory 'sql/test/BugTracker-2018` itself:


# 10:14:58 >  
# 10:14:58 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=34466" "--set" "mapi_usock=/var/tmp/mtest-10982/.s.monetdb.34466" "--set" "monet_prompt=" "--forcemito" "--dbpath=/ufs/sjoerd/@Monet-stable/var/MonetDB/mTests_sql_test_BugTracker-2018" "--set" "embedded_py=true"
# 10:14:58 >  

# builtin opt 	gdk_dbpath = /ufs/sjoerd/@Monet-stable/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 34466
# cmdline opt 	mapi_usock = /var/tmp/mtest-10982/.s.monetdb.34466
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbpath = /ufs/sjoerd/@Monet-stable/var/MonetDB/mTests_sql_test_BugTracker-2018
# cmdline opt 	embedded_py = true
# cmdline opt 	gdk_debug = 553648138

# 10:14:58 >  
# 10:14:58 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-10982" "--port=34466"
# 10:14:58 >  


# 10:14:59 >  
# 10:14:59 >  "Done."
# 10:14:59 >  

